src/controllers/controllerpresetfilehandler.cpp

```diff
@@ -111,16 +110,27 @@ void ControllerPresetFileHandler::addScriptFilesToPreset(
 
     // Build a list of script files to load
     QDomElement scriptFile = controller.firstChildElement("scriptfiles")
-            .firstChildElement("file");
+                                     .firstChildElement("file");
 
     // Default currently required file
-    preset->addScriptFile(REQUIRED_SCRIPT_FILE, "", true);
+    preset->addScriptFile(REQUIRED_SCRIPT_FILE,
+            "",
+            QFileInfo(systemPresetsPath.absoluteFilePath(REQUIRED_SCRIPT_FILE)),
+            true);
```

Comment:
This prevents loading a modified common-controller-scripts.js from the user folder. I think we should handle this the same way as other scripts (preset path first, then fallback to system), just do it automatically without having to specify it explicitly in the preset.